src/main/webapp/paginationScript.js

```diff
@@ -61,21 +61,33 @@ function displayCards(listAugment) {
       //Card being appended to the resultsContent div
       let cardToAppend = resultsCardsArray[i];
 
+      //The actual image element to which the image src will be applied
+      let resultsImageElement = locateImageElement(cardToAppend.card);
+
       if (cardToAppend.photoReference != 'none') {
-        //The actual image element to which the image src will be applied
-        let imageElement = cardToAppend.card.children[0].children[0];
-        loadImage(imageElement, cardToAppend.photoReference);
+        loadImage(resultsImageElement, cardToAppend.photoReference);
       }
 
       resultsContent.appendChild(cardToAppend.card);
     }
   }
 }  
 
+function locateImageElement(card) {
+  let cardChildren = card.childNodes;
+  let child = 0;
+
+  while (cardChildren[child].className != 'results-image') {
+    child++;
+  }
+
+  return cardChildren[child].children[0];
```

Comment:
Can you grab this by the "img" tag instead? 

This imageElement is a div (that contain many elements). If another element gets added before the 'img' tag, this will likely break. getElementsByTagName seems like it should help here.

Reply:
Done!

Reply:
I knew there was a better way than that but forgot that existed